pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -337,3 +375,52 @@ func gitRefInfo(repo git.Repository, dir, ref string) (buildapi.GitRefInfo, erro
 	info.Message = lines[4]
 	return info, nil
 }
+
+// WaitForBuildComplete waits for a build identified by the name to complete
+func WaitForBuildComplete(c osclient.BuildInterface, name string) error {
```

Comment:
@kargakis you can use --wait or --follow or both... In case you use both, then yeah, the code runs twice on server side and in client. However, if you just use --wait the code runs just in CLI (and the logs are not streamed out). So I think this PR is correct.